src/hooks/ctsApiSupport/__tests__/useCtsApi.test.js

```diff
@@ -1,46 +1,72 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { render } from '@testing-library/react';
+import { act, render, screen } from '@testing-library/react';
 import { useCtsApi } from '../useCtsApi';
 import { useAppSettings } from '../../../store/store';
-import { getClinicalTrialsQuery } from '../../../services/api/actions/getClinicalTrialsQuery';
+import { getClinicalTrialsAction } from '../../../services/api/actions/getClinicalTrialsAction';
 import { getClinicalTrials } from '../../../services/api/clinical-trials-search-api/getClinicalTrials';
 
 jest.mock('../../../store/store');
 jest.mock(
 	'../../../services/api/clinical-trials-search-api/getClinicalTrials.js'
 );
 
+/* eslint-disable react/prop-types */
+const UseCtsApiSupportSample = ({ actions }) => {
+	const { loading, payload, error, aborted } = useCtsApi(actions);
+
+	const i = 10;
```

Comment:
NIT:  not needed

Reply:
Thanks -- this was where I put my breakpoint to view those vars above. It is removed.